File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int lastIndex = 0;
        int lastNum = nums[0];
        for (int i=0; i < nums.length; i++) {
            if (nums[i] != lastNum){
                lastIndex++;
                nums[lastIndex]=nums[i];
                lastNum=nums[i];
            }
        }
        return lastIndex+1;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        l = len(nums)
        i, j = 0, 0
        p = nums[0]
        while i < l:
            if nums[i]!=p:
                j+=1
                nums[j]=nums[i]
                p=nums[i]
            i+=1
        return j+1